.github/workflows/aws-deploy.yaml

```diff
@@ -45,7 +45,9 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Generate AWS CDK bootstrap template
-        run: npm run generate-cdk-bootstrap-template
+        run: npx cdk bootstrap --json --show-template > org-formation/200-baseline/tmptemplate.json
```

Comment:
the actual cdk command should be in package.json, look for `generate-cdk-bootstrap-template`

`tmptemplate.json` should not be added to the repo so please add it to `.gitignore`

Reply:
OK